freeipa.spec.in

```diff
@@ -344,6 +345,7 @@ Requires: libsss_autofs
 Requires: autofs
 Requires: libnfsidmap
 Requires: nfs-utils
+Requires: python-icalendar >= 3.9.2
 Requires(post): policycoreutils
```

Comment:
Please provide required version of python-icalendar into @freeipa/freeipa-master copr, and create rebase request for fedora rawhide